### PR TITLE
Added proto readme

### PR DESCRIPTION
**Pull request by jontje**
As per title

### CODE REVIEW

File: proto/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Information
+
+* [egm.proto](egm.proto): Created by [ABB Robotics](https://new.abb.com/products/robotics) for specifying the EGM communication protocol.
```
Comment (gavanderhoorn) [inline proto/README.md]: If this file was downloaded from an ABB site, perhaps we should provide a URL.

If it was copied from a manual, perhaps we should add a document identifier or title+page nr.

Comment (jontje) [reply proto/README.md]: I got the file from the installation folder of RobotWare. I will add a description of how to find it.

**Comment by gavanderhoorn** [comment]
Let's wait on Travis and then merge.

**Comment by jontje** [comment]
I have rebased this branch as well since GitHub indicated that it was outdated

**Pull request merged**